x-pack/plugins/observability_shared/public/components/page_template/search_bar_portal.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { ReactNode, useEffect, useMemo } from 'react';
+import ReactDOM from 'react-dom';
+import { EuiPanel } from '@elastic/eui';
+import { createHtmlPortalNode, InPortal, OutPortal } from 'react-reverse-portal';
+export interface Props {
+  children: ReactNode;
```

Comment:
You should then make `children` optional?
```suggestion
  children?: ReactNode;
```

Reply:
Done